FE/src/components/UserInfo.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function UserInfo ({token, adminPriv, userData}) {
+    // console.log(userData);
+    // console.log(adminPriv);
+    return (
+        <div className = 'userInfo'>
+            <img src={userData.imgurl} alt={`User account image for ${userData.username}`} height="15%" width="22.5%"/>
+            <h2>{userData.username}</h2>
+            {token !== null && 
```

Comment:
I don't know if this was the code's intention, but this userPersonal element will currently render on any user page as long as the user is logged in. The token condition just verifies that the user is logged in, not that the user matches the user page